kr.yaml

```diff
@@ -11,7 +11,7 @@ months:
   1:
   - name: Korean New Year
     regions: [kr]
-    function: lunar_to_solar(year, month, day)
+    function: lunar_to_solar(year, month, day, :ko)
```

Comment:
Could we make this `:kr` so it matches the existing region symbol? I raised this over in the ruby repo as well.